android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -78,9 +78,13 @@ public String getName() {
     }
 
     @ReactMethod
-    public void initialize(String publishableKey) {
-        Radar.initialize(getReactApplicationContext(), publishableKey);
-        Radar.setReceiver(receiver);
+    public void initialize(String publishableKey, boolean fraud) {
+        if (fraud) {
+            Radar.initialize(getReactApplicationContext(), publishableKey, receiver, Radar.RadarLocationServicesProvider.GOOGLE, fraud);
+        } else {
+            Radar.initialize(getReactApplicationContext(), publishableKey);
+            Radar.setReceiver(receiver);
```

Comment:
Hmm, unless I'm missing something, I think we want to set the receiver regardless of whether or not `fraud: true`

Reply:
I could remove the `if/else` and collapse this to `Radar.initialize(getReactApplicationContext(), publishableKey, receiver, Radar.RadarLocationServicesProvider.GOOGLE, fraud)`. Just didn't want to touch existing `initialize()` logic if not necessary.

Reply:
Way of doing that while also setting the receiver if fraud is true would be:
```
if (fraud) {
    Radar.initialize(getReactApplicationContext(), publishableKey, receiver, 
    Radar.RadarLocationServicesProvider.GOOGLE, fraud);
Radar.setReceiver(receiver);
} else { 
    Radar.initialize(getReactApplicationContext(), publishableKey);
    Radar.setReceiver(receiver);
}

Reply:
Do we think the separate `setReceiver()` call is needed if `receiver` is passed to `initialize()`?

Reply:
Cool